web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -153,16 +152,7 @@ odoo.define("web_drop_target", function (require) {
          */
         _onBodyFileDragover: function (ev) {
             ev.preventDefault();
-            const actionManager = this.findAncestor(function (ancestor) {
-                return ancestor instanceof ActionManager;
-            });
-            const controller = actionManager.currentDialogController;
```

Comment:
@etobella We need your input here. If I compare Runboat of current 14.0 with Runboat of this PR, and in both I install CRM and open the page of any random `res.partner`, then purely functionally, on 14.0 the dropzone does not appear on drag and this PR fixes that. Upon Javascript debugging, I see that in the 14.0 case, `actionManager.currentDialogController` is `null`. So this whole code does not get executed.

@KKamaa claims this is since #2299 but to me it seems that it's introduced in #2066 instead. Since you're the one that proposed that, could you give your opinion on removing these checks in the way that was done here? Maybe there's a simpler solution to the problem?

Reply:
(in other words, functional review OK, but I'm also a bit confused about the code change)